tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -553,7 +553,7 @@ When('the user batch declines these shares using the webUI', async function(file
     await client.page.FilesPageElement.filesList().toggleFileOrFolderCheckbox('enable', item[0])
   }
 
-  return client.page.personalPage().declineAllCheckedShares()
+  return client.page.sharedWithMePage().batchDeclineShares()
```

Comment:
Well found. This looks like it was an existing error, not related to the other timing problems.

Reply:
Yes, that was weird. Was either hard failing, or had an error message in the console while succeeding. That was an unrelated flaky test and I took the chance to solve it. 😎

Reply:
(If I remember correctly I introduced it myself 😁 )